app/controllers/alaveteli_pro/plans_controller.rb

```diff
@@ -1,6 +1,7 @@
 # -*- encoding : utf-8 -*-
-# Does not inherit from AlaveteliPro::BaseController as is pre-login
-class AlaveteliPro::PlansController < ApplicationController
+class AlaveteliPro::PlansController < AlaveteliPro::BaseController
+  skip_before_action :pro_user_authenticated?
+  skip_before_action :set_in_pro_area, only: [:index]
```

Comment:
Actually I think now that we can do this, we should as per [original intent](https://github.com/mysociety/alaveteli-professional/issues/330#issuecomment-326596057).

(Nothing you've done wrong here :) – I just didn't have `set_in_pro_area` available previously, but makes sense to use the pro livery now that we inherit from `AlaveteliPro::BaseController`.)